### DISCUSSION TITLE
npm create tamagui@latest on select is printing list

### DISCUSSION THREAD

**Discussion by benschac** [tamagui/tamagui#1935] [General]
### Current Behavior

![CleanShot 2023-12-02 at 08 35 53@2x](https://github.com/tamagui/tamagui/assets/2502947/510e2651-19d6-4df8-b8e6-8f63b990f629)

### Expected Behavior

should be able to select item with out repeating content

### Tamagui Version

```markdown
latest
```

### Platform (Web, iOS, Android)

```markdown
cli
```

### Reproduction

```markdown
npm create tamagui@latest
```

**Comment by natew** [comment]
This happens when your terminal is too narrow, not sure how we fully fix but maybe other libraries are better? Open to input here.
